### PR TITLE
Updated wound frames

### PR DESCRIPTION
**Pull request by Lemon-King**
Created 3 additional wound interfaces from mehuge's wounds.
Requires cu-components pull for correct functionality: [Pull Request](https://github.com/CUModSquad/cu-components/pull/3)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/cu-ui/pull/15" target="_blank">ReviewNinja

File: character/package.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   "version": "0.1.0",
-  "description": "Camelot Unchained UI - character",
+  "description": "Camelot Unchained UI - Wounds/Player",
   "license": "MPL-2.0",
   "main": "src/ts/main.tsx",
-  "author": "mehuge@sorcerer.co.uk",
-  "contributors": [],
+  "author": "City State Entertainment <support@citystateentertainment.com>",
+  "contributors": [
+    "mehuge@sorcerer.co.uk"
```
Comment (codecorsair) [inline character/package.json:9]: add yourself! :smile:

File: character/src/ts/main.tsx [hunk 2]
```diff
@@ -1,58 +1,54 @@
...
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 
 /// <reference path="../tsd/tsd.d.ts" />
+
 import * as React from 'react';
-import * as Reflux from 'reflux';
+import * as cuAPI from 'cu-core';
```
Comment (codecorsair) [inline character/src/ts/main.tsx:10]: delete this line.  You're not using the import additionally, you should not import anything as cuAPI that's a reserved variable name for our client -> ui object.

**Comment by codecorsair** [comment]
Update to use the new camelot-unchained lib and resubmit for review.

**Pull request closed without merging**